README.md

```diff
@@ -1,2 +1,5 @@
 # past_project
-performing a code review for a past project
+performing a code review on a past project
```

Comment:
- [] [OPTIONAL] Let's make sure all points are capitalized. This way your project will be more professional.